python/lsst/sims/photUtils/matchUtils.py

```diff
@@ -138,13 +138,9 @@ class matchStar(matchBase):
     the format of those included here.
     """
 
-    def __init__(self, sEDDir = None, kuruczDir = None, mltDir = None, wdDir = None):
+    def __init__(self, kuruczDir = None, mltDir = None, wdDir = None):
 
         """
-        @param [in] sEDDir is a place to specify a different path to a directory that follows the same
-        directory structure as SIMS_SED_LIBRARY. For instance, a different version of the LSST
-        SIMS_SED_LIBRARY.
-
         @param [in] kuruczDir is a place to specify a different path to kurucz SED files than the
         files in the LSST sims_sed_library. If set to None it will default to the LSST library.
         Will probably be most useful for those who want to use loadGalfast without downloading the
```

Comment:
I can make the quick fix now and I can open a new JIRA ticket assigned to me to organize this into a non-specific read-in code. The reason I wrote it that way originally was to make it easier for the readGalfast code to use it's information to match the galfast stars to a specific SED type. But we only end up matching by colors in this code anyway so I should be able to rewrite this to have a general read-in method and adjust for that in the readGalfast code as well.